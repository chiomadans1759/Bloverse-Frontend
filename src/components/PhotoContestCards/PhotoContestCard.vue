<template>
  <main class="card text-white photoContestCard">
    <img src="@/assets/photo-contest-1.jpg" class="card-img" alt="trend image">
    <div class="card-img-overlay photoContestCard--overlay">
      <div class="content photoContestCard--overlay__content">
        <p class="card-text font-weight-bold photoContestCard--text">
          Lorem ipsum dolor sit amet, vel accumsan
        </p>
        <p class="card-text photoContestCard--username">
          <a class="btn btn-primary initials" href="#" role="button"></a>
          <span class="name">John Doe</span>
        </p>
      </div>
    </div>
  </main>
</template>

<script>
export default {
  name: 'Trend'
}
</script>

<style lang='scss' scoped>
@import './photocontestcard.scss';
</style>
