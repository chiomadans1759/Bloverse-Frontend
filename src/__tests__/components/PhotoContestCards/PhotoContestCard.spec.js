import { mount, RouterLinkStub  } from '@vue/test-utils'
import PhotoContestCard from '@/components/PhotoContestCards/PhotoContestCard.vue';

describe('PhotoContestCard Component', () => {
  it('should render correctly', () => {

    const wrapper = mount(PhotoContestCard, {
      stubs: {
        RouterLink: RouterLinkStub
      }
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
