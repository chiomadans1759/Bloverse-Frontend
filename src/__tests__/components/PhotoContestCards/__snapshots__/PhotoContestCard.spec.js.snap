// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoContestCard Component should render correctly 1`] = `
<main
  class="card text-white photoContestCard"
>
  <img
    alt="trend image"
    class="card-img"
    src="@/assets/photo-contest-1.jpg"
  />
   
  <div
    class="card-img-overlay photoContestCard--overlay"
  >
    <div
      class="content photoContestCard--overlay__content"
    >
      <p
        class="card-text font-weight-bold photoContestCard--text"
      >
        
        Lorem ipsum dolor sit amet, vel accumsan
      
      </p>
       
      <p
        class="card-text photoContestCard--username"
      >
        <a
          class="btn btn-primary initials"
          href="#"
          role="button"
        />
         
        <span
          class="name"
        >
          John Doe
        </span>
      </p>
    </div>
  </div>
</main>
`;
